test/e2e-test-application/e2e/tests/0-fiddle/fiddle-navigation.spec.js

```diff
@@ -366,6 +366,43 @@ describe('Fiddle', () => {
 
         cy.contains('Login again');
       });
+      it('User settings in profile menu with custom label', () => {
+        newConfig.settings = {
+          userSettings: {
+            userSettingsProfileMenuEntry: {
+              label: 'My UserSettings',
+              icon: 'settings'
+            }
+          }
+        }
+        visitLoggedInWithAuthConfig('/', newConfig);
+        cy.get('[data-testid="luigi-topnav-profile"] button').click();
+        cy.get('[data-testid="settings-link"]').should('exist');
+        cy.get('[data-testid="settings-link"]').contains('My UserSettings')
+      });
+      it('User settings in profile menu with default label', () => {
+        newConfig.settings = {
+          userSettings: {}
+        }
+        visitLoggedInWithAuthConfig('/', newConfig);
+        cy.get('[data-testid="luigi-topnav-profile"] button').click();
+        cy.get('[data-testid="settings-link"]').should('exist');
+        cy.get('[data-testid="settings-link"]').contains('Settings')
+      });
+      it.only('TESTS', () => {
+        newConfig.navigation.profile = {
+          logout: {
+            label: 'Bye bye',
+            icon: 'sys-cancel'
+          }
+        };
+        newConfig.auth.disableAutoLogin = false;
+        visitLoggedInWithAuthConfig('/', newConfig);
+
+        cy.get('[data-testid="luigi-topnav-profile"] button').click();
+        logoutLink().should('exist');
+        cy.get('[data-testid="settings-link"]').should('not.exist');
```

Comment:
When I run `npm run e2e:open` it executes only the last tests called "TESTS". Could you please enable the rest of the tests to be executed and maybe rename it to "Testing the profile without settings config" or so?

Reply:
Agreed, I assume `only` was the leftover. Please remove it.

Reply:
Definitly true.. thanks for finding. I forgot to remove it.